<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEphemerisDataDialog</class>
 <widget class="QWidget" name="AddEphemerisDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>462</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>星历数据参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="6" column="0">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>偏心率：</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>平近点角(度)：</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>升交点赤经(度)：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>倾角(度)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>卫星名称：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>瞬跟历元日期：</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>近地点幅角(度)：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>半长轴：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>任务代号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="satelliteIdentification"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="satelliteName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>卫星标识：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>瞬跟历元时刻：</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="satellitCode"/>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateOfEpoch"/>
      </item>
      <item row="4" column="1">
       <widget class="QTimeEdit" name="timeOfEpoch"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="semimajorAxis"/>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="eccentricity"/>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="dipAngle"/>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="rightAscension"/>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="argumentOfPerigee"/>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="meanAnomaly"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirmBtn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
